com.unity.render-pipelines.universal/Runtime/ScriptableRenderer.cs

```diff
@@ -1588,13 +1590,12 @@ static void SetRenderTarget(CommandBuffer cmd, RTHandle[] colorAttachments, RTHa
         internal virtual void EnableSwapBufferMSAA(bool enable) { }
 
         [Conditional("UNITY_EDITOR")]
-        void DrawGizmos(ScriptableRenderContext context, Camera camera, GizmoSubset gizmoSubset)
+        void DrawGizmos(ScriptableRenderContext context, Camera camera, GizmoSubset gizmoSubset, CommandBuffer cmd)
```

Comment:
should we pass RenderingData instead of cmd?

Reply:
fixed in 3939959